### PR TITLE
Remove build section from js guidelines

### PR DESCRIPTION
**Pull request by RichardLitt**
This is "completely out of date", according to @diasdavid in https://github.com/ipfs/community/issues/225#issuecomment-272419143. I remove it here. What should go in its place?

### CODE REVIEW

**Comment by diasdavid** [comment]
It needs to mention that users can get a pre-bundled version by doing `npm run build`,

**Comment by RichardLitt** [comment]
@diasdavid Is that only for js-ipfs, or for all projects? Does that mean that contributors need to add an `npm run build` to each project, too (and use gulp, and so on)?

**Comment by diasdavid** [comment]
`npm run build` is a npm script we add to the package.json, but in fact what it is doing is calling `aegir-build`, which itself calls `gulp build` [:D](http://aguidetogettinglifewrong.com/wp-content/uploads/2015/05/Russian-Dolls.jpg)

It is available for every project that uses `aegir`

**Comment by RichardLitt** [comment]
@diasdavid Added section.

File: js-project-guidelines.md [hunk 1]
```diff
@@ -336,6 +337,10 @@ We suggest either of these:
 ]
 ```
 
+##### Building
+
+You can get a pre-bundled version by running `npm run build`, an npm script we add to the `package.json`. This calls `aegir-build`, which itself calls `gulp build`. This is available for every project that uses `aegir`.
```
Comment (diasdavid) [inline js-project-guidelines.md]: image

just realised now that I never submitted this CR :(

**Pull request merged**